guides/deployment/docker.md

```diff
@@ -0,0 +1,304 @@
+# Introduction to Deployments with docker
+
+First things first, we need a running application to deploy it. If you don't yet have one, head to
+[Up and Running Guide](up_and_running.html).
+
+Docker enables you to deploy to the platform of your choice AWS, Google cloud, Digital ocean...
+
+In this guide we will cover a simple deployement to a single machine. There are two steps for that.
```

Comment:
s/deployement/deployment/

Reply:
I would also add a semicolon here “There are two steps for that:”

Reply:
when you say semicolon, I think you mean colon :-)